### DISCUSSION TITLE
Language settings are reverted upon closing browser

### DISCUSSION THREAD

**Discussion by ypsilonkah** [open-webui/open-webui#3398] [General]
# Bug Report

## Description

**Bug Summary:**
Language settings in UI are reverted when closing the browser

**Steps to Reproduce:**
Open settings, select a language and save. Close browser and reopen. Login and default language reverted to english.

**Expected Behavior:**
Any selected language should persist

**Actual Behavior:**
Language is reverted to english.

## Environment

- **Open WebUI Version:** 0.3.5
- **Ollama (if applicable):** not applicable

- **Operating System:** Ubuntu 24.04 (as server for open webui); Windows 11 as client
- **Browser (if applicable):** Firefox 127.0.1

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [x] I have included the Docker container logs.

## Logs and Screenshots

**Docker Container Logs:**
```
INFO:     Application startup complete.

INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)

/app

  ___                    __        __   _     _   _ ___ 
 / _ \ _ __   ___ _ __   \ \      / /__| |__ | | | |_ _|
| | | | '_ \ / _ \ '_ \   \ \ /\ / / _ \ '_ \| | | || | 
| |_| | |_) |  __/ | | |   \ V  V /  __/ |_) | |_| || | 
 \___/| .__/ \___|_| |_|    \_/\_/ \___|_.__/ \___/|___|
      |_|                                               

      

v0.3.5 - building the best open-source AI user interface.

https://github.com/open-webui/open-webui

INFO:apps.openai.main:get_all_models()

INFO:apps.ollama.main:get_all_models()

INFO:apps.openai.main:get_all_models()

INFO:apps.ollama.main:get_all_models()

ERROR:apps.ollama.main:Connection error: Cannot connect to host 192.168.178.31:11434 ssl:default [No route to host]

ERROR:apps.ollama.main:Connection error: Cannot connect to host 192.168.178.31:11434 ssl:default [No route to host]

INFO:     connection open

...

INFO:     connection closed
```
(I am aware of the connection error; the ollama server is not booted atm. Problem persists when the server is present)

## Installation Method
Installation via docker-compose.

**Comment by tjbck** [comment]
Unable to reproduce in dev, if anyone manages to reproduce PR welcome!

**Comment by ypsilonkah** [comment]
The 0.3.7 update partly solved the problem. The language is now (as described in the change notes) selected according to the browser settings, which is the language I wanted to set. However, when I select English as the standard language and close/reopen the browser German is the chosen language.

**Reply by tjbck** [reply]
Make sure you're on the latest!
